docs/authentication/social-connections/apple.mdx

```diff
@@ -24,8 +24,9 @@ Enabling OAuth via [Sign in with Apple](https://developer.apple.com/sign-in-with
 
 ## Configure for your development instance
 
-To make the development flow as smooth as possible, Clerk uses preconfigured shared OAuth credentials and redirect URIs for _development instances_.
-For **web based flows**, no other configuration is needed. For **native sign-in flows**, you must provide your app's [Bundle ID](https://developer.apple.com/documentation/appstoreconnectapi/bundle_ids).
+For _development instances_, Clerk uses preconfigured shared OAuth credentials and redirect URIs. For **web based flows**, no other configuration is needed. For **native sign-in flows**, you must provide your app's [Bundle ID](https://developer.apple.com/documentation/appstoreconnectapi/bundle_ids).
+
+To configure your development instance, follow these steps:
 
 1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections).
 1. In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**.
```

Comment:
but then an unknowing user will click every tab at the top, and search the entire page looking for "User & Authentication". until they reach the fourth and final tab.
so I don't agree with this - I think because we don't use screenshots, and because we now have four entirely separate contexts, we should be very explicit when directing the user to what context they should be looking at ("configure").